JMS/com/ibm/mq/samples/jms/JwtHelper.java

```diff
@@ -0,0 +1,85 @@
+/*
+* (c) Copyright IBM Corporation 2024
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package com.ibm.mq.samples.jms;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.util.logging.Logger;
+import java.net.http.HttpResponse;
+import org.json.JSONObject;
+import com.ibm.mq.samples.jms.SampleEnvSetter;
+
+public class JwtHelper {
+    private static final Logger logger = Logger.getLogger("com.ibm.mq.samples.jms");
+    private static String tokenEndpoint = "";
```

Comment:
why static properties, why not instance properties?

Reply:
These properties are expected to remain same for every instance of JwtHelper.

Reply:
In that case they should be set once, in the first time a constructor is called, protected by an if ! empty block so they only get set once, and all instances of JwtHelper use them.